compiler/optimizer/OMRSimplifierHandlers.cpp

```diff
@@ -11997,6 +11998,216 @@ TR::Node *sxorSimplifier(TR::Node * node, TR::Block * block, TR::Simplifier * s)
    return node;
    }
 
+template <typename T>
+static TR::Node *expandbitsSimplifier(TR::Node *node, TR::Block *block, TR::Simplifier *s)
+   {
+   s->simplifyChildren(node, block);
+
+   TR::Node *srcChild = node->getFirstChild();
+   TR::Node *maskChild = node->getSecondChild();
+
+   if (maskChild->getOpCode().isLoadConst())
+      {
+      T mask = maskChild->getConst<T>();
+      if (srcChild->getOpCode().isLoadConst() &&
+          performTransformation(s->comp(), "%sReduced %s of const src and const mask in node [%s] to const\n",
```

Comment:
It may be more beneficial if someone tries to find out what transformed to print the node pointer (See other perform transformation usage).